### ISSUE TITLE
4.2.4 Step 2.4: Identify Technologies Relied Upon

### ISSUE THREAD

**Issue by mgifford** [w3c/wai-wcag-em#126]
Accessibility isn't skin deep. The examples in [4.2.4 Step 2.4: Identify Technologies Relied Upon](https://w3c.github.io/wai-wcag-em/#step2d),  "HTML, CSS, JavaScript, SVG, WAI-ARIA, and PDF" are all front-end. Also none of them are talking about the libraries that make the site. 

Folks may skip over the NOTE, or see it as an optional nice-to-have. It shouldn't be optional. 

If it is Drupal or WordPress it matters in terms of how the bugs are going to be addressed.  Is it React or Vanilla JS, it will matter as to how the issues are addressed. 

One of the big advantages of WCAG-EM is that it is machine-readable content. Having a Wappalyzer type of site analyzer or just having the internal team document the technology use would be useful. Especially as sometimes you want to actually report the issue not in your own codebase, but upstream. You want the issue to be fixed and not just patched locally. 

An organization with many sites should be able to use WCAG-EM to notice that all sites using this technology have common problems. WCAG EM should be actively supporting the identification of frameworks (both front-end and back-end).

**Comment by hidde** [comment]
The common thread in the technologies mentioned (HTML, CSS, JavaScript, SVG, WAI-ARIA, and PDF) is that they are standards, we've chosen this abstraction level on purpose for longevity and to remain framework/tool agnostic. 

In 4.2.4 we do encourage mentioning authoring tools and design systems relied upon, but here too, we don't include names of specific CMSes or design systems.

So while it would absolutely be valid for folks following the methodology to go into detail about their technology use in their report, people do that in some reports today already, the methodology doc itself stays away from that.

**Comment by hidde** [comment]
Agreed that this info is helpful, we are listing these in [2.4 Identify technologies relied upon](https://w3c.github.io/wai-wcag-em/#step2d) (also encouraging authoring tools and design systems to be listed).
